boards/arm/lpcxpresso54114/pinmux.c

```diff
@@ -23,7 +23,7 @@ static int lpcxpresso_54114_pinmux_init(struct device *dev)
 		device_get_binding(CONFIG_PINMUX_MCUX_LPC_PORT1_NAME);
 #endif
 
-#ifdef CONFIG_UART_MCUX_FLEXCOMM_0
+#if DT_HAS_NODE(DT_NODELABEL(flexcomm0)) && defined(CONFIG_UART_MCUX_FLEXCOMM)
```

Comment:
is there a macro to check the compat on flexcomm0?

Reply:
`DT_NODE_HAS_COMPAT(DT_NODELABEL(flexcomm0), nxp_lpc_usart)`

Reply:
ok, then we should use it here and everywhere else we set up flexcomm pins

Reply:
Done